src/components/NewTodoForm/NewTodoForm.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './NewTodoForm.css';
+
+export class NewTodoForm extends React.Component {
+  state={
+    username: 'Choose a user',
+    title: '',
+    errortitle: false,
+    errorusername: false,
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+    const re = /[^\w ]+/;
+
+    this.setState({
+      [name]: value.replace(re, ''),
+      [`error${name}`]: false,
+    });
+  }
+
+  handleSubmit = (event) => {
+    const { title, username } = this.state;
+    const { onAdd, users } = this.props;
+
+    event.preventDefault();
+
+    if (!title.trim()) {
+      this.setState({
+        errortitle: true,
+      });
+    }
+
+    if (username === 'Choose a user') {
+      this.setState({
+        errorusername: true,
+      });
+    }
+
+    if (title && username !== 'Choose a user') {
+      const foundUser = users.find(user => user.name === username);
+
+      this.setState({
+        title: '',
+        username: 'Choose a user',
+      });
+
+      onAdd(title, foundUser);
+    }
+  }
+
+  render() {
+    const { username, errorusername, title, errortitle } = this.state;
+    const { users } = this.props;
+
+    return (
+      <div className="form">
+        <form
+          onSubmit={this.handleSubmit}
+        >
+          <div className="form__input form-group">
+            <label htmlFor="username" className="font-weight-bold">
+              Username
+            </label>
+            <select
+              name="username"
+              id="username"
+              className="form-control"
+              value={username}
+              onChange={this.handleChange}
+            >
+              <option>Choose a user</option>
+              {users.map(user => (
+                <option
+                  key={user.id}
+                  value={user.name}
+                >
+                  {user.name}
+                </option>
+              ))}
+            </select>
+
+            {errorusername
+              && <p className="text-danger">Please, choose a user</p>
```

Comment:
```suggestion
              && <p className="error">Please, choose a user</p>
```

Reply:
This className is from bootstrap. So it is better to  leave it as it is.